app/models/course/condition/achievement.rb

```diff
@@ -37,7 +38,9 @@ def achievement_conditions(achievement)
         (SELECT cca.achievement_id
           FROM course_condition_achievements cca INNER JOIN course_conditions cc
             ON cc.actable_type = 'Course::Condition::Achievement' AND cc.actable_id = cca.id
-            WHERE cc.conditional_id = #{achievement.id}) ids
+            WHERE cc.conditional_id = #{conditional.id}
```

Comment:
Is it this [one](http://makandracards.com/makandra/1675-using-heredoc-for-prettier-ruby-code)?